modin/engines/dask/pandas_on_dask/frame/axis_partition.py

```diff
@@ -21,13 +21,13 @@
 
 
 class PandasOnDaskFrameAxisPartition(PandasFrameAxisPartition):
-    def __init__(self, list_of_blocks, bind_ip=False):
+    def __init__(self, list_of_blocks, get_ip=False):
         # Unwrap from BaseFramePartition object for ease of use
         for obj in list_of_blocks:
             obj.drain_call_queue()
         self.list_of_blocks = [obj.future for obj in list_of_blocks]
-        if bind_ip:
-            self.list_of_ips = [obj.ip for obj in list_of_blocks]
+        if get_ip:
+            self.list_of_ips = [obj._ip_cache for obj in list_of_blocks]
```

Comment:
Is it ok if the cache is empty?

Reply:
The cache will be valid value after any remote call (any operation on a Modin DataFrame).